AutomatedTesting/Gem/PythonTests/prefab/Prefab_Operations.py

```diff
@@ -0,0 +1,190 @@
+"""
```

Comment:
I'm not a fan in general that every prefab operation has test results associated to them.
This adds overhead to every time we attempt to do a prefab operation, if we have a test that instantiates 50 prefabs there will be tons of results in the console.

Instead, I recommend that the checks and result reporting to be done on the specific test that tests that specific functionality.

Reply:
Note that you can also use `assert False, "My error"` instead of results for the function if you prefer

Reply:
I think have these check is quite useful to capture undesired behaviors in various test cases. But you also make a good point about the test that instantiates 50 prefabs. If we want to add test that instantiates multiple prefabs later I will prefer adding an operation for doing that with fewer checks.

Reply:
Use assert at several places for now.